src/xcode/ENA/HealthCertificateToolkit/Sources/HealthCertificateToolkit/DigitalGreenCertificateFake.swift

```diff
@@ -8,7 +8,7 @@ import SwiftCBOR
 
 enum DigitalGreenCertificateFake {
 
-    public static func makeBase45Fake(from certificate: DigitalGreenCertificate, and header: CBORWebTokenHeader) -> Result<Base45, CertificateDecodingError>  {
```

Comment:
Needs to be public. We are using this in our ENA testing.

Reply:
But then the `enum` needs to be public as it's `internal` per default. I was wondering why Swiftlint complained but it still worked that way 🤷‍♂️ My guess is, that's because it's not a separate repository and still part of the ENA project.

Just to mention why the `enum` get's public in a few seconds.

Reply:
Yes true. I needed to make it public (also other stuff too) on another branch, where the tests are in place. You can leave it internal. Depending which PR is faster, the other PR needs to be adjusted 😅